src/app/data-planes/data/index.ts

```diff
@@ -76,7 +77,19 @@ export type MeshGatewayDataplane = PartialMeshGatewayDataplane & {
   listenerEntries: MeshGatewayListenerEntry[]
   routePolicies: Meta[]
 }
-
+const DataplaneOutbound = {
+  fromObject(item: PartialDataplaneOutbound): DataplaneOutbound {
+    return {
+      ...item,
+      name: item.tags['kuma.io/service'],
```

Comment:
**Question**: Why do we need `name` _and_ `service`?

Reply:
We don't use this right now but in the same way that we fill out what we know at this point and append it later for inbounds this would do the same. See for gateway inbounds, we use the name of the service for the name and then once we know they port (later on in the `*View.vue`) we append that onto the end of the name.

Reply:
But why do we need two properties with the same value?

Reply:
Its not the same value, in some cases we append the port to it later.

Reply:
Where does this happen? And do we actually mutate the DataplaneOverview object for that or is it just happening in place somewhere?